_api-reference/cat/cat-repositories.md

```diff
@@ -36,7 +36,7 @@ In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-refe
 
 Parameter | Type | Description
 :--- | :--- | :---
-local | Boolean | Whether to return information from the local node only instead of from the cluster_manager node. Default is false.
+local | Boolean | Whether to return information from the local node only instead of from the cluster_manager node. Default is `false`.
```

Comment:
```suggestion
local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is `false`.
```